exporter/src/main.rs

```diff
@@ -143,8 +143,7 @@ fn take_screenshot(
                     .renderer_mut()
                     .downcast_mut::<WgpuRenderBackend<TextureTarget>>()
                     .unwrap();
-                // Use straight alpha
-                renderer.capture_frame(false)
+                renderer.capture_frame()
```

Comment:
Are these changes mistakenly included in this PR?

Reply:
Actually no. I started with fixing a warning using `#[cfg(not(target_family = "wasm"))]`, and got carried away a bit...

I can split these changes to a separate PR if it makes sense.